cirq/ops/common_gates.py

```diff
@@ -349,8 +348,8 @@ class MeasurementGate(raw_types.Gate,
         key: The string key of the measurement.
         invert_mask: A list of values indicating whether the corresponding
             qubits should be flipped. The list's length must not be longer than
-            the number of qubits, but it is permitted to be shorted. Qubits with
-            indices past the end of the mask are not flipped.
+            the number of qubits, but it is permitted to be shorted.
+    Qubits with indices past the end of the mask are not flipped.
```

Comment:
Why has this been moved out of the invert_mask description? It's describing invert_mask.

Reply:
What has been moved? `Qubits with`? It's still describing `invert_mask`.

Reply:
The indentation now indicates it is about the method instead of about the parameter.